private/buf/cmd/buf/command/alpha/repo/reposync/reposync.go

```diff
@@ -150,13 +152,17 @@ func run(
 	} else if flags.Create {
 		return appcmd.NewInvalidArgumentErrorf("--%s is required if --%s is set.", createVisibilityFlagName, createFlagName)
 	}
+	if flags.Remote == "" {
+		return appcmd.NewInvalidArgumentErrorf("--%s cannot be empty.", remoteFlagName)
+	}
```

Comment:
This layer has the only check against empty remote. If we ever decide to support local syncs again, it's just a matter of fixing this layer.

Reply:
Removed, by default we sync from local.